scaffold/nightwatch/sites.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// phpcs:ignoreFile
```

Comment:
New syntax for me. I always used `@codingStandardsIgnore` and `@codingStandardsIgnore<Start,End>`. There any difference?

Otherwise this makes total sense to me and is good once builds are passing.

Reply:
Yes, I had to [double-check](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders) which is the "new" vs. "old" syntax:
> Note: Before PHP_CodeSniffer version 3.2.0, use // @codingStandardsIgnoreFile instead of // phpcs:ignoreFile. The `@codingStandards` syntax is deprecated and will be removed in PHP_CodeSniffer version 4.0.